.takt/facets/instructions/validate-gap.md

```
@@ -0,0 +1,43 @@
+要件と既存コードベースのギャップを分析し、実装戦略の情報を提供せよ。
```

markdownlint-cli2 (0.21.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)